qemu/tests/cfg/dump_guest_core.cfg

```diff
@@ -0,0 +1,18 @@
+- dump_guest_core:
+    type = dump_guest_core
+    virt_test_type = qemu
+    start_vm = no
+    pre_command = "ulimit -c unlimited"
+    pre_command1 = "echo "/var/core.%p" > /proc/sys/kernel/core_pattern"
+    trigger_core_dump_command = "kill -s SIGSEGV %s"
+    gdb_command = "gdb /var/core.%s --command=%s"
+    gdb_command_file = "/home/gdb_command"
+    crash_script = "/home/crash.cmd"
+    extra_params += " -machine dump-guest-core=%s"
+    x86_64:
+        extra_params += " -device vmcoreinfo"
+    variants:
```

Comment:
Can you give some comments about "on" or "off" here,for example, if the guest memory is included or not

Reply:
When 'dump-guest-core=off' is specified, guest memory is omitted from the core dump.

Reply:
Please add the comment in the cfg, it will be much better.